core/src/container.cpp

```diff
@@ -189,7 +189,7 @@ void ContainerBase::add(Stage::pointer&& stage) {
 
 bool ContainerBase::insert(Stage::pointer&& stage, int before) {
 	if (!stage) {
-		ROS_ERROR_STREAM(name() << ": received invalid stage pointer");
+		RCLCPP_ERROR_STREAM(rclcpp::get_logger("ContainerBase"), name() << ": reveived invalid stage pointer");
```

Comment:
This should use a static logger (per class?) together with [`get_child()`](https://docs.ros2.org/foxy/api/rclcpp/classrclcpp_1_1Logger.html#a68ab28459973c1072d3ad7a593ac994e) imo